app/controllers/wikis_controller.rb

```diff
@@ -1,5 +1,9 @@
 class WikisController < ApplicationController
   def index
+      if ! current_user.present? || current_user.standard?
+          @wikis = Wiki.where(private: false)
+          return
```

Comment:
You don't ever need to 'return' in Ruby, unless you are existing the method early. The last value of the method is what gets returned :) In this case, it would be @wikis.